jenkins/Jenkinsfile-1

```diff
@@ -0,0 +1,12 @@
+pipeline {
+  agent {

```

Comment:
I didn't test this file as part of the steps.  Was I supposed to?

Reply:
no this is a sample jenkinsfile I can remove it